content/kubernetes-platform-services/artifact-storage/examples/chart-repo-helmfile.yaml

```diff
@@ -0,0 +1,134 @@
+repositories:
```

Comment:
should we get the code from `geodesic` on GitHub?

Reply:
No. We provide the custom helmfile that is focused on chart-repo helmfile.
This is the pattern we use for all backing services. Probably we will change it in future.
For now, let's leave this for consistency documentation.